code-challenges.js

```diff
@@ -14,28 +14,103 @@ const fruit2 = "banana"
+//use test variables
+// Expected output: "42 below boil", "350 above boil", "212 @boil"
+const findBoil = (addTemp)=>{
+    //start with most strigent option
+    if(addTemp === 212){
+        return `${addTemp} is at boiling point`
+    } else if (addTemp < 212){
+    //second smallest option
+        return `${addTemp} is below boiling point`  
+    }else{
+    //This is the catch all because there is an infinite number thats larger
+        return `${addTemp} is above boiling point`
+    }
+}
+console.log(findBoil(temp1));
+console.log(findBoil(temp2));
+console.log(findBoil(temp3));
+console.log(findBoil(temp4));
+
 
 
-// --------------------2) Create the code that will combine the two arrays and return the length using the variables provided below. Expected output: 10
 
+// --------------------2) Create the code that will combine the two arrays and return the length using the variables provided below. Expected output: 10
+// Combine two arrays using .concat
+// Make new Variable to put the values of both arrays
+// Use .length on new variable
+//count Array .length is an informational command that returns the number of elements in the array.
 const myNumbers1 = [3, 7, 0, 36, -9]
 const myNumbers2 = [8, -7, 42, 9, 13]
 
+var comboArrays = myNumbers1.concat(myNumbers2)
 
-// --------------------3) Create the code that will reverse the letters of a string using the test variable provided below. Expected output: "1202 ohcE"
+console.log(comboArrays.length);
 
-const currentCohort = "Alpha 2022"
 
+// --------------------3) Create the code that will reverse the letters of a string using the test variable provided below. Expected output: "1202 ohcE"
+// NOTE: As far as I know there is no reverse method for str.
+//1. Use the split() method to return a new array
+const currentCohort = "Alpha 2022"
+var arrCohort = currentCohort.split("")
+//2. Use the reverse() method to reverse the create array
+var reverseArray = arrCohort.reverse()
+//3. Use joint() method to join all elements of the array into a str.
+var joinArray = reverseArray.join("")
+console.log(joinArray);
+// Ill try a  function with a for loop
+const reverseString =(str)=>{
+    //because functions are reusable I'll create an empty str
+    var currentCohort2 = ""
+    //Im not totally clear why '-1' works instead of .length but this is what i got to work
+    //As long as i is greater than or equal 0, the loop will go on
+    // we decrement i after each iteration
+    for(let i = str.length-1;i>=0;i--){
+        //str.length =10; i = str.length -1 and 'newString' =newString + str[i]
+        //first iteration i =10-1=9   newStr. =""+"2"="2"
+        //2nd iteration i =9-1=8;     newStr. ="2"+"2" = "22"
+        //3rd iteration i =8-1=7;     newStr. ="22"+"0" ="220"
+        //4th iteration i =7-1=6;     newStr. ="220"+"2"="2202" etc.
+        currentCohort2 += str[i]
+    }
+    //return reverse str.
+    return currentCohort2;
+}
+console.log(reverseString("Alpha 2022"));
```

Comment:
Loving that you decided to build this out using a function as well!  This is a great way to practice and explore other approaches.  In regards to the '-1' in your for loop, this strategy is starting from the last index and iterating backward until you reach the first index.  .length determines the total length of the string which differs from the index number of the last instance.  The index of the last value in this particular string would be 9 since the index starts on 0.